### DISCUSSION TITLE
Node-module

### DISCUSSION THREAD

**Discussion by deveshsnair** [tailwindlabs/tailwindcss#7380] [Help]
Hi my node-modules folder is getting bigger day by day it over 10MBs now is it possible to reduce of completely remove this folder from the directory..?

I can't push the project for production due to this issue.

**Comment by alexanderbluhm** [comment]
@deveshsnair Is this related to Tailwind CSS? I don't think so 😅

Nevertheless, the common approach is to add `node_modules` to your **.gitignore** file so it is not pushed to your repository.
In your deployment you then execute `npm install` to install the packages on the server.

**Reply by deveshsnair** [reply]
Thanks alot
